pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -153,13 +166,15 @@ func (b *BridgePodInterface) discoverPodNetworkInterface() error {
 		return err
 	}
 
-	// Get interface MAC address
-	mac, err := Handler.GetMacDetails(podInterface)
-	if err != nil {
-		log.Log.Reason(err).Errorf("failed to get MAC for %s", podInterface)
-		return err
+	if len(b.vif.MAC) == 0 {
```

Comment:
ok. i see where the concern is. if some network plugin manages the mac addresses somehow.
why did we originally move the mac from the pod interface to the VM? couldn't we just send an arp notifying the IP address now belong to a new mac?

Reply:
No idea really. :) @vladikr @phoracek any back story behind retaining MAC address in the initial podinterface.go implementation?

Reply:
We just wanted to have a generic solution that doesn't rely on the CNI network provider. Some providers use DHCP to hand out IPs, some just automagically set it. We wanted to be sure.

I'm not 100% confident that this solution will work with all providers, nor whether it will work over a long period of time (if the vm restarts after the original lease from CNI DHCP is expired) - however, I know that we have another solution that worked so far.
I'm comfortable enough to merge this.